### PR TITLE
Add mock man and amend card accordingly

### PR DESCRIPTION
**Pull request by varun13169**
image

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -1,17 +1,19 @@
...
-	<!--
+  <head>
+    <meta charset="utf-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
+    <meta name="theme-color" content="#000000" />
+    <!--
       manifest.json provides metadata used when your web app is added to the
```
Comment (zoef-yt) [inline public/index.html]: Do remove extra commented code.

File: src/assets/WishlistHeart.jsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      className="dui-card-prod-hzntl__wishlist-btn_svg dui-util-spc-pad-0_8rem-xs icon icon-tabler icon-tabler-heart"
+      width="56"
+      height="56"
+      viewBox="0 0 24 24"
+      // stroke-width="1.5"
+      // fill="#F34E4E"
+      // stroke-linecap="round"
+      // stroke-linejoin="round"
```
Comment (zoef-yt) [inline src/assets/WishlistHeart.jsx]: Remove the commented code if not needed.

File: src/pages/productListingPage/productListingPage.jsx [hunk 3]
```diff
@@ -1,25 +1,50 @@
...
+  const { wishlist, setWishlist } = useWishlist();
   const { productAndFilterState, setProductAndFilterState } =
     useProductFilter();
 
   useEffect(() => {
+    // Fetch Products
     (async function () {
       const { data } = await axios.get("/api/products");
-      setProducts(data.products);
+      // setProducts(data.products);
```
Comment (zoef-yt) [inline src/pages/productListingPage/productListingPage.jsx]: remove commented code.

File: src/pages/productListingPage/productListingPage.jsx [hunk 4]
```diff
@@ -33,7 +58,18 @@ export default function ProductListingPage() {
...
+                  cart,
+                  setCart,
+                  wishlist,
+                  setWishlist,
+                })}
+              />
+            );
           })}
           {/* <div className="mobl-fltr-btn-helper"></div>
             <button className="mobl-fltr-btn reset-button-inherit-parent" href="/pages/cartPage/cartPage.html">
```
Comment (zoef-yt) [inline src/pages/productListingPage/productListingPage.jsx]: remove commented code.

**Review by zoef-yt — CHANGES_REQUESTED** [review]

**Comment by zoef-yt** [comment]
Please add some description to the PR, and do the suggested changes.

File: src/App.jsx [hunk 5]
```diff
@@ -1,10 +1,15 @@
...
+import { ProductListingPage } from "./pages";
+import { Route, Routes } from "react-router-dom";
+import MockAPI from "./mockman/MockAPI";
 
 export default function App() {
   return (
     <div className="App">
-        <ProductListingPage></ProductListingPage>
+      <Routes>
+        <Route path="/products" element={<ProductListingPage />}></Route>
```
Comment (humzahasan) [inline src/App.jsx]: Suggestion : Route tag are self closing according to the docs.

**Review by humzahasan — APPROVED** [review]
- Code looks Clean. 
- Good to see comments before each function call.

Good to merge!

**Comment by santhoshpemmaka** [comment]
* code looking good.
*  If possible remove comments in .jsx files.
Good to merge!

File: public/index.html (hunk 1 above)
Comment (varun13169) [reply public/index.html]: Done.

File: src/assets/WishlistHeart.jsx (hunk 2 above)
Comment (varun13169) [reply src/assets/WishlistHeart.jsx]: Done.

File: src/pages/productListingPage/productListingPage.jsx (hunk 3 above)
Comment (varun13169) [reply src/pages/productListingPage/productListingPage.jsx]: Done.

**Review by santhoshpemmaka — APPROVED** [review]

File: src/pages/productListingPage/productListingPage.jsx (hunk 4 above)
Comment (varun13169) [reply src/pages/productListingPage/productListingPage.jsx]: Done

File: src/App.jsx (hunk 5 above)
Comment (varun13169) [reply src/App.jsx]: Done.

**Review by zoef-yt — APPROVED** [review]

**Pull request merged**